apps/strapi-cms/src/components/common/hero-slide.json

```diff
@@ -9,6 +9,10 @@
     "title": {
       "type": "string"
     },
+    "subhead" : {
+      "type": "blocks",
+      "required": false
+    },
```

Comment:
To avoid build errors, I think you should separate this PR. This would consent us to merge the changes to Strapi before the FE pull request